backend/src/main/java/com/htecgroup/skynest/util/JwtUtils.java

```diff
@@ -38,9 +39,14 @@ public class JwtUtils {
   public static Algorithm ALGORITHM;
 
   public static String generate(
```

Comment:
like in other cases, you can create a separate method for role claim. For example generateRoleClaim method that will call this generic method , that way "roles" as claim will be inside that new method and you will have control what is sent as claim instead of giving the option for free text

Reply:
Will try to do something like that